week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -31,13 +31,33 @@ var parkAvenueHouse = {
 */
 
 // returns the full name (first name + last name) of the owner of the house
-function getOwnerFullName(house) {}
+function getOwnerFullName(house) {
+  return house.currentOwner.firstName+ ' ' + house.currentOwner.lastName;
+}
 
 // returns an array of the owners' email addresses of the two houses
-function getEmailAddresses(house1, house2) {}
+function getEmailAddresses(house1, house2) {
+  var arr = [];
+  //arr[0] = house1.currentOwner.email; 
+  //arr[1] = house2.currentOwner.email;
+  arr.push(house1.currentOwner.email);//at the end of the array

```

Comment:
Nice way of doing it. Everything before this in week 4 has been good by the way - you've aced it so far, so I've had no comments.